src/cascadia/TerminalSettingsModel/Profile.cpp

```diff
@@ -80,165 +73,77 @@ void Profile::DeleteUnfocusedAppearance()
-
-    profile->_Deleted = source->_Deleted;
-    profile->_Guid = source->_Guid;
-    profile->_Name = source->_Name;
-    profile->_Source = source->_Source;
-    profile->_Hidden = source->_Hidden;
-    profile->_Icon = source->_Icon;
-    profile->_CloseOnExit = source->_CloseOnExit;
-    profile->_TabTitle = source->_TabTitle;
-    profile->_TabColor = source->_TabColor;
-    profile->_SuppressApplicationTitle = source->_SuppressApplicationTitle;
-    profile->_UseAcrylic = source->_UseAcrylic;
-    profile->_AcrylicOpacity = source->_AcrylicOpacity;
-    profile->_ScrollState = source->_ScrollState;
-    profile->_Padding = source->_Padding;
-    profile->_Commandline = source->_Commandline;
-    profile->_StartingDirectory = source->_StartingDirectory;
-    profile->_AntialiasingMode = source->_AntialiasingMode;
-    profile->_ForceFullRepaintRendering = source->_ForceFullRepaintRendering;
-    profile->_SoftwareRendering = source->_SoftwareRendering;
-    profile->_HistorySize = source->_HistorySize;
-    profile->_SnapOnInput = source->_SnapOnInput;
-    profile->_AltGrAliasing = source->_AltGrAliasing;
-    profile->_BellStyle = source->_BellStyle;
-    profile->_ConnectionType = source->_ConnectionType;
-    profile->_Origin = source->_Origin;
-
-    // Copy over the font info
-    const auto weakRefToProfile = weak_ref<Model::Profile>(*profile);
-    winrt::com_ptr<FontConfig> sourceFontInfoImpl;
-    sourceFontInfoImpl.copy_from(winrt::get_self<FontConfig>(source->_FontInfo));
-    auto copiedFontInfo = FontConfig::CopyFontInfo(sourceFontInfoImpl, weakRefToProfile);
-    profile->_FontInfo = *copiedFontInfo;
-
-    // Copy over the appearance
-    winrt::com_ptr<AppearanceConfig> sourceDefaultAppearanceImpl;
-    sourceDefaultAppearanceImpl.copy_from(winrt::get_self<AppearanceConfig>(source->_DefaultAppearance));
-    auto copiedDefaultAppearance = AppearanceConfig::CopyAppearance(sourceDefaultAppearanceImpl, weakRefToProfile);
-    profile->_DefaultAppearance = *copiedDefaultAppearance;
-
-    if (source->_UnfocusedAppearance.has_value())
+    for (const auto& sourceProfile : source)
     {
-        Model::AppearanceConfig unfocused{ nullptr };
-        if (source->_UnfocusedAppearance.value() != nullptr)
-        {
-            // Copy over the unfocused appearance
-            winrt::com_ptr<AppearanceConfig> sourceUnfocusedAppearanceImpl;
-            sourceUnfocusedAppearanceImpl.copy_from(winrt::get_self<AppearanceConfig>(source->_UnfocusedAppearance.value()));
-            auto copiedUnfocusedAppearance = AppearanceConfig::CopyAppearance(sourceUnfocusedAppearanceImpl, weakRefToProfile);
-
-            // Make sure to add the default appearance as a parent
-            copiedUnfocusedAppearance->InsertParent(copiedDefaultAppearance);
-            unfocused = *copiedUnfocusedAppearance;
-        }
-        profile->_UnfocusedAppearance = unfocused;
+        target.emplace_back(sourceProfile->CopyInterned(visited));

```

Comment:
```suggestion
        target.emplace_back(std::move(sourceProfile->CopyInterned(visited)));
```
Maybe?

Reply:
Both `std::move` as well as function expressions like `sourceProfile->CopyInterned()` return a value of `rvalue` category. `rvalue`s bind to `T&&` arguments.
In other words: A function expression returns a value that's being moved, so you don't need to call `std::move` on it.